Updated apiDefinition.swagger.json

gulshankhurana opened the pull request:
- Changed the name from 'FCA' to 'Financial Conduct Authority UK' and also updated the description.
- Fixed the 'Common Search' action.
- Added 'Find Firm Details' and 'Find Product Details' actions.

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

<img width="179" alt="Con-Def" src="https://github.com/microsoft/PowerPlatformConnectors/assets/67651578/d0995593-6613-4daf-826b-cc251d245fe5">

<img width="520" alt="Con-Test-CommonSearch-Individual" src="https://github.com/microsoft/PowerPlatformConnectors/assets/67651578/214f25d0-9e38-4fd5-a363-a94ff14430d5">

<img width="519" alt="Con-Test-CommonSearch-Fund" src="https://github.com/microsoft/PowerPlatformConnectors/assets/67651578/4894bd8e-c00b-46e7-8869-caac31131047">

<img width="511" alt="Con-Test-CommonSearch-Firm" src="https://github.com/microsoft/PowerPlatformConnectors/assets/67651578/2cb0f85c-4654-4524-a95f-92321a696f04">

<img width="513" alt="Con-Test-FirmDetails" src="https://github.com/microsoft/PowerPlatformConnectors/assets/67651578/ca3483a1-29bc-4438-b463-ac4ede847f7c">

<img width="520" alt="Con-Test-ProductDetails" src="https://github.com/microsoft/PowerPlatformConnectors/assets/67651578/e04d9e66-b6ca-42e7-a64e-65330058f446">

<img width="487" alt="Flow-CommonSearch-Individual" src="https://github.com/microsoft/PowerPlatformConnectors/assets/67651578/f17c0e85-55d5-470d-81c0-853e230b370d">

<img width="494" alt="Flow-CommonSearch-Firm" src="https://github.com/microsoft/PowerPlatformConnectors/assets/67651578/53e92283-0a0d-4cbf-a719-7987043388bc">

<img width="480" alt="Flow-CommonSearch-Fund" src="https://github.com/microsoft/PowerPlatformConnectors/assets/67651578/07b4f99c-d011-4a51-b0f0-d605bc6ad744">

<img width="324" alt="Flow-FindFirmDetailsByFRN" src="https://github.com/microsoft/PowerPlatformConnectors/assets/67651578/512620c3-bf82-4cb9-943a-b26467aae23d">

<img width="329" alt="Flow-FindProductDetailsByPRN" src="https://github.com/microsoft/PowerPlatformConnectors/assets/67651578/9043483e-8dcf-43f9-9534-0f0dccceafca">

gulshankhurana committed:
Create ReadME.md

Created Financial Conduct Authority (FCA) UK Connector folder and ReadME file underneath

gulshankhurana committed:
Update ReadME.md

gulshankhurana committed:
Update ReadME.md

gulshankhurana committed:
Update ReadME.md

gulshankhurana committed:
Rename ReadME.md to ReadME.md

Corrected the foldername

gulshankhurana committed:
Update ReadME.md

gulshankhurana committed:
Added files via upload

Uploaded the following files:
apiDefinition.swagger.json
apiProperties.json

gulshankhurana committed:
Update apiProperties.json

gulshankhurana committed:
Merge branch 'microsoft:dev' into dev

gulshankhurana committed:
Update ReadME.md

gulshankhurana committed:
Added file via upload

Added the following files:
* apiDefinition.swagger.json
* apiProperties.json

gulshankhurana committed:
Delete independent-publisher-connectors/FInancial Conduct Authority (FCA) UK Connector/ReadME.md

gulshankhurana committed:
Delete independent-publisher-connectors/FInancial Conduct Authority (FCA) UK Connector/apiDefinition.swagger.json

gulshankhurana committed:
Delete independent-publisher-connectors/FInancial Conduct Authority (FCA) UK Connector/apiProperties.json

gulshankhurana committed:
Rename ReadME.md to ReadME.md

gulshankhurana committed:
Delete independent-publisher-connectors/FInancial Conduct Authority (FCA) UK Connector/ReadME.md

gulshankhurana committed:
Added ReadME.md file

gulshankhurana committed:
Rename ReadME.md to ReadME.md

gulshankhurana committed:
Add files via upload

gulshankhurana committed:
Delete independent-publisher-connectors/FInancial Conduct Authority (FCA) UK Connector directory

gulshankhurana committed:
Merge branch 'microsoft:dev' into dev

gulshankhurana committed:
Update apiDefinition.swagger.json

Updated the title and description

gulshankhurana committed:
Merge pull request #1 from gulshankhurana/gulshankhurana-patch-2

Update apiDefinition.swagger.json

gulshankhurana commented:
@vmanoharas, I have updated the apiDefinition.swagger.json as per your advice under #3242

vmanoharas commented on independent-publisher-connectors/Financial Conduct Authority (FCA) UK Connector/apiDefinition.swagger.json:
```diff
@@ -1,8 +1,8 @@
   "info": {
-    "title": "FCA",
-    "description": "The online Financial Services Register is a public record of financial services firms, individuals and other bodies regulated by the Financial Conduct Authority (FCA)",
+    "title": "Financial Conduct Authority (FCA) UK",
```
The 'title' value goes over the character limit '30' ValuePath: /info/title. Kindly fix this.

gulshankhurana replied:
@vmanoharas , Fixed as advised. Thanks.

vmanoharas reviewed (commented):
Hello @gulshankhurana, 

Kindly fix the review comments.

gulshankhurana committed:
Merge branch 'microsoft:dev' into dev

gulshankhurana committed:
Update apiDefinition.swagger.json

Truncated the title to keep it 30 characters long

gulshankhurana committed:
Update apiDefinition.swagger.json

Truncated the title length to 30 characters

gulshankhurana committed:
Merge pull request #2 from gulshankhurana/gulshankhurana-patch-3

Update apiDefinition.swagger.json

gulshankhurana commented:
@vmanoharas , fixed the issue as advised. Thanks.

vmanoharas reviewed (dismissed):
Hello @gulshankhurana 
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks.
Please let us know if you have any questions.
Thank you very much for working with us.

vmanoharas commented:
[[certify-connector]]

vmanoharas commented:
[[certify-connector]]

gulshankhurana committed:
Merge branch 'microsoft:dev' into dev

gulshankhurana committed:
Update ReadME.md

* Fixed 'Common Search' action 
* Added 'Find Firm Details' action
* Added 'Find Product Details' action

gulshankhurana committed:
Update apiProperties.json

Updated apiProperties.json file

gulshankhurana committed:
Update apiDefinition.swagger.json

Updated apiDefinition.swagger.json

gulshankhurana committed:
Update apiDefinition.swagger.json

gulshankhurana commented:
Hi @vmanoharas
I have fixed action errors caused by change in api definition by the api provider. I also added 2 more actions.
Please approve at your earliest convenience as some people are eagerly awaiting these changes to take effect so that they can start using this connector asap.

Kind regards
Gulshan

vmanoharas reviewed (approved):
Hello @gulshankhurana 
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks.
Please let us know if you have any questions.
Thank you very much for working with us.

vmanoharas commented:
[[certify-connector]]

vmanoharas commented:
Hello @gulshankhurana, 

This PR is failing to upload into our validation tools, kindly close this PR and create new fresh PR.

gulshankhurana committed:
Merge branch 'microsoft:dev' into dev

gulshankhurana commented:
Hi @vmanoharas 

As suggested, I have opened a new PR #3495 . 

Thanks.
Gulshan
